x/gamm/pool-models/balancer/amm_joinpool_test.go

```diff
@@ -80,6 +81,7 @@ var calcSingleAssetJoinTestCases = []calcJoinSharesTestCase{
 		poolAssets:   oneTrillionEvenPoolAssets,
 		tokensIn:     sdk.NewCoins(sdk.NewInt64Coin("uosmo", 50_000)),
 		expectShares: sdk.NewInt(2_499_999_968_750),
+		expectToleranceError: false,
```

Comment:
No need to specify `expectToleranceError: false,` since non-initialized bool is equivalent to false in Go